test/storage/index.js

```diff
@@ -0,0 +1,116 @@
+const { config, run } = require('hardhat');
+const { expect } = require('chai');
+const fs = require('fs');
+const os = require('os');
+const path = require('path');
+
+const extractStorageLayout = require(path.join(config.paths.root, 'scripts/extract-storage-layout'));
+
+describe('STORAGE LAYOUT TESTS', function () {
```

Comment:
good work on separating this in its own test.

The evaluation does not need dynamic on-chain data.